web/views.py

```diff
@@ -39,6 +39,8 @@ def compare(request):
         concept_query_string = escape(strip_tags(request.GET.get('concept', '')))
         lang1_query_string = escape(strip_tags(request.GET.get('lang1', '')))
         lang2_query_string = escape(strip_tags(request.GET.get('lang2', '')))
+        meta_structures = json.loads(meta_data)["structures"]
+        concept_friendly_name = list(meta_structures.keys())[list(meta_structures.values()).index(concept_query_string + '.json')]
```

Comment:
Would this just be easier to have the concept `id` instead of `id.json`? Because the meta data file is easy enough to correct and so would the above line to simplify it a bit